src/Event/CommandHandlers/CopyEventHandler.php

```diff
@@ -42,11 +46,13 @@ public function handle($command): void
 
         // Add the new event to the same production as the original one.
         // This is not done as part of Event::copy() because productions are not event-sourced.
-        try {
-            $production = $this->productionRepository->findProductionForEventId($originalEventId);
-            $this->productionRepository->addEvent($newEventId, $production);
-        } catch (EntityNotFoundException $e) {
-            return;
+        if ($this->copyProduction) {
```

Comment:
I missed this feature completely with the first review. This would have been something for a separate pull request. Also, the name is very confusing, also see https://github.com/cultuurnet/appconfig/pull/719/files#r1686143312